File: School.Domain/Entities/Unit.cs
using System.Collections;
using System.Collections.Generic;
using System;

namespace School.Domain.Entities
{
    public class Unit
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public List<Room> Rooms { get; set; }
        public List<CheckingAccount> CheckingAccounts { get; set; }
        public List<Student> Students { get; set; }
        //public List<UnitTeacher> Teachers { get; set; }
        public int AddressId { get; set; }
        public Address Address { get; set; }
        public int ContactId { get; set; }
        public Contact Contact { get; set; }
    }
}

File: School.WebApi/Views/Default/src/app/organization/Entities/Unit.ts
import { Room } from './Room';
import { CheckingAccount } from './CheckingAccount';
import { Address } from './Address';
import { Student } from './Student';
import { Teacher } from './Teacher';
import { Contact } from './Contact';

export class Unit {
    id: number;
    name: string;
    addressid: number;
    address: Address;
    contactid: number;
    contact: Contact;
    rooms: Room[];
    checkingaccounts: CheckingAccount[];
    students: Student[];
                                                                                                                                                          
}
